core/io/file_access_network.cpp

```diff
@@ -184,7 +184,7 @@ Error FileAccessNetworkClient::connect(const String &p_host, int p_port, const S
 	ERR_FAIL_COND_V_MSG(err != OK, err, "Cannot connect to host with IP: " + String(ip) + " and port: " + itos(p_port));
 	while (client->get_status() == StreamPeerTCP::STATUS_CONNECTING) {
 		//DEBUG_PRINT("trying to connect....");
-		OS::get_singleton()->delay_usec(1000);
+		OS::get_singleton()->delay_usec(6900);
```

Comment:
Does this change anything? It's a tight loop during connect, if anything, it should cause longer startup time when using the remote FS.

Reply:
It will reduce CPU substantially while the client is in the connecting state (which can take quite a few seconds on slow connections I.e. Mobile).

While this performance hit is usually not an issue, it will save battery life on mobile devices which create multiple simultaneous connections.

I feel like `144 hz` is a more reasonable default than `1000 hz` for most operations where we only wait for something rather than running the main loop.

Reply:
You prefer I revert this line?

Reply:
> It will reduce CPU substantially while the client is in the connecting state (which can take quite a few seconds on slow connections I.e. Mobile).

I honestly don't see this being a problem. `FileAccessNetworkClient` is not designed for distribution, but for debugging, and this will just make the most common case (debugging over local network/in a controlled environment) slower, in exchange of a minor battery life improvement of your debug devices.

Reply:
Ok I've reverted this one